src/components/dialogs/create-case-dialog.js

```diff
@@ -88,10 +93,7 @@ export function CreateCaseDialog({ onClose, open }) {
                 <FormattedMessage id="ImportNewCase" />
             </DialogTitle>
             <DialogContent>
-                <DialogContentText>
-                    <FormattedMessage id="createNewCaseDescription" />
-                </DialogContentText>
-                <div>{nameField}</div>
+                {nameField}
```

Comment:
Since nameField seems to be a component, wouldn't it be logical to capitalise it ?

Reply:
fixed